manage.py

```diff
@@ -2,27 +2,9 @@
 import os
 import sys
 
-# Now we can import from third-party libraries.
 from django.core.management import execute_from_command_line
 
-# Don't try to force a new setting module on us
-if 'DJANGO_SETTINGS_MODULE' not in os.environ:
-
-    # use settings_test.py for running tests
-    if 'test' in sys.argv:
-        settings_mod = 'settings_test'
-    else:
-        # or try the optional "local" settings module first
-        try:
-            import settings_local  # noqa
-        except ImportError:
-            # or use the plain settings.py module
-            settings_mod = 'settings'
-        else:
-            settings_mod = 'settings_local'
-
-    # override the env var with what we want
-    os.environ['DJANGO_SETTINGS_MODULE'] = settings_mod
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'kuma.settings.local')
```

Comment:
I'm very against adding random shell scripts, since we've in the past had trouble maintaining them (or writing good ones in the first place). We used fabric for a while, but that had similar problems. I'm a fan of well written Makfiles since they hide complexity well enough. Lemme update the django-tests Make task then to accept parameters.